kinto/core/resource/__init__.py

```diff
@@ -922,7 +922,8 @@ def _extract_filters(self, queryparams=None):
                 )
                 continue
 
-            m = re.match(r'^(min|max|not|lt|gt|in|exclude)_([\w\.]+)$', param)
+            allKeywords = '|'.join([i.name.lower() for i in COMPARISON])
```

Comment:
I felt it's naive to have this regex listing exactly what is in `Comparison` enum.
Only downside is that it also includes `eq_` as a prefix, which surely should be fine?